prisma/schema.prisma

```diff
@@ -67,3 +68,20 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+// Profile schema
+model Profile {
+    id            Int     @id @default(autoincrement())
+    user          User    @relation(fields: [userId], references: [id])
+    userId        String  @unique
+    preferredName String?
+    phoneNumber   String?
+    bio           String?
+    role          UserRole @relation(fields: [roleId], references: [id])
+    roleId        String @default("MENTEE")
+}
+
+model UserRole {
+    id String @id
```

Comment:
I think this is okay to have the role itself be the id. It smells a little funny but I think it is okay. The other thing I would think to do would be name id "roleName" or something. I think what you wrote is good, let's go with it. If it goes wrong at least I can tell myself I told you so.